Makefile

```diff
@@ -7,7 +7,7 @@ all: mod inst gen build spell lint test
 
 .PHONY: ci
 ci: ## CI build pipeline
-ci: all diff
+ci: all check diff
```

Comment:
How about addint it to `all` (e.g. before `test`)?

Reply:
I did not put it there because it is quite slow to run it, and the output might not change that often.
I can put it in the test recipe if you prefer 😄

Reply:
>  it is quite slow to run it

Then let's keep as it is.

Reply:
May be not renaming it, but adding a precommit like this could do the trick:

```Makefile
all: precommit

ci: precommit

precommit: lint test check....
```

WDYT?